contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -22,10 +24,16 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
   uint256 internal constant PERFORM_GAS_MIN = 2_300;
   uint256 internal constant CANCELLATION_DELAY = 50;
   uint256 internal constant PERFORM_GAS_CUSHION = 5_000;
-  uint256 internal constant REGISTRY_GAS_OVERHEAD = 80_000;
   uint256 internal constant PPB_BASE = 1_000_000_000;
   uint64 internal constant UINT64_MAX = 2**64 - 1;
   uint96 internal constant LINK_TOTAL_SUPPLY = 1e27;
+  // L1_FEE_DATA_PADDING includes 35 bytes for L1 data padding for Optimism
+  bytes public L1_FEE_DATA_PADDING = "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff";
+  // MAX_INPUT_DATA represents the estimated max size of the sum of L1 data padding and msg.data in performUpkeep
+  // function, which includes 4 bytes for function selector, 32 bytes for upkeep id, 35 bytes for data padding, and
+  // 32 bytes for estimated perform data
```

Comment:
Let's make this 64. e.g. in this tx https://optimistic.etherscan.io/tx/0x2feb9fab2cac60b995b5b4d81c63e871552aa986aa595a8390946610c302d632 performData takes two words. (Some specifics of abi encoding that I don't fully understand)

Reply:
Done